sandbox/index.js

```diff
@@ -68,6 +71,15 @@ if (BABYLON.Engine.isSupported()) {
                 currentScene.activeCamera.keysRight.push(69); // E
                 currentScene.activeCamera.keysRight.push(68); // D
             }
+            defaultCamera = currentScene.activeCamera;
+            arcRotateCamera = new BABYLON.ArcRotateCamera("camera", 4.71238898038469, 1.5707963267948966, 3, BABYLON.Vector3.Zero(), currentScene);

```

Comment:
I'm wondering if the radius could be detected.
Like the same way we do this: https://github.com/BabylonJS/Babylon.js/blob/master/src/babylon.scene.ts#L3467